### PR TITLE
sys-firmware/intel-microcode: Move to portage-stable

### PR DESCRIPTION
**Pull request by sayanchowdhury**
# sys-firmware/intel-microcode: Sync with Gentoo upstream; updates to 20220809

## Testing done
~Jenkins CI passed http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/6376/cldsv/~
Jenkins CI running http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/6497/cldsv/

To be merged with https://github.com/flatcar/portage-stable/pull/361

- [x] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)

### CODE REVIEW

**Review by krnowak — COMMENTED** [review]
I just went through entire git history of this directory for find out what's the reason for adding `/${PVR}` to the `SLOT` variable. The earliest additions just has `- Add PVR to sub-slot` in commit message, without any explanation. This is not helpful.

Do you know if this modification is really needed? If not, maybe we should just move the package to portage-stable.

**Comment by sayanchowdhury** [comment]
> I just went through entire git history of this directory for find out what's the reason for adding `/${PVR}` to the `SLOT` variable. The earliest additions just has `- Add PVR to sub-slot` in commit message, without any explanation. This is not helpful.
> 
> Do you know if this modification is really needed? If not, maybe we should just move the package to portage-stable.

The reason for adding the SLOT was mentioned in the PR https://github.com/coreos/coreos-overlay/pull/3196

**Comment by krnowak** [comment]
> > I just went through entire git history of this directory for find out what's the reason for adding `/${PVR}` to the `SLOT` variable. The earliest additions just has `- Add PVR to sub-slot` in commit message, without any explanation. This is not helpful.
> > Do you know if this modification is really needed? If not, maybe we should just move the package to portage-stable.
> 
> The reason for adding the SLOT was mentioned in the PR [coreos/coreos-overlay#3196](https://github.com/coreos/coreos-overlay/pull/3196)

Thanks!

> 
> Maybe we should add the reasoning into the ebuild? `This will rebuild coreos-kernel whenever microcode is updated, and ensures that coreos_production_image_packages.txt includes intel-microcode.`

Either ebuild or just in commit message - that would help, yes. But on the other hand - when do we need this? We do full rebuilds now.

**Comment by sayanchowdhury** [comment]
> 
> Either ebuild or just in commit message - that would help, yes. But on the other hand - when do we need this? We do full rebuilds now.

I'm thinking the same, if we even need this. A good iteration would be to move to portage-stable and if we face issues then we can revert back?

**Comment by krnowak** [comment]
> > Either ebuild or just in commit message - that would help, yes. But on the other hand - when do we need this? We do full rebuilds now.
> 
> I'm thinking the same, if we even need this. A good iteration would be to move to portage-stable and if we face issues then we can revert back?

Sounds good.

**Comment by sayanchowdhury** [comment]
I've moved the PR to draft PR. The Jenkins build is running with the move to portage. Once passed, I'll open the PR for review again.

File: changelog/updates/2022-08-29-intel-microcode-20220809_p20220809.md [hunk 1]
```diff
@@ -0,0 +1 @@
+- intel-microcode ([20220809](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/releases/tag/microcode-20220809))
```
Comment (dongsupark) [inline changelog/updates/2022-08-29-intel-microcode-20220809_p20220809.md:1]: You would then probably want to delete the added files from `changelog/updates` and `changelog/security`.

**Comment by sayanchowdhury** [comment]
Jenkins CI passed.

**Review by krnowak — APPROVED** [review]

**Pull request merged**